test/e2e/apimachinery/crd_conversion_webhook.go

```diff
@@ -358,25 +358,37 @@ func deployCustomResourceWebhookAndService(f *framework.Framework, image string,
 func verifyV1Object(crd *apiextensionsv1.CustomResourceDefinition, obj *unstructured.Unstructured) {
 	gomega.Expect(obj.GetAPIVersion()).To(gomega.BeEquivalentTo(crd.Spec.Group + "/v1"))
 	hostPort, exists := obj.Object["hostPort"]
-	framework.ExpectEqual(exists, true)
```

Comment:
isn't there something like `framework.ExpectTrue(exists, "host port should exist")` ? These are oneliners intentionally to not bloat the tests.

Reply:
Hi @sttts, we currently do not have `ExpectTrue` yet in our testing framework so I followed #105678 and used `Failf` instead.

https://github.com/kubernetes/kubernetes/blob/62d9f8ba80f4cc660a88dcc34f56c5c6f7df17ea/test/e2e/framework/expect.go#L24